src/Uno.UI/UI/Xaml/Media/Animation/KeyTime.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Windows.UI.Xaml.Media.Animation
 {
-	public partial struct KeyTime : IEquatable<KeyTime>
+	public partial struct KeyTime : IEquatable<KeyTime>, IComparable<KeyTime>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [When implementing IComparable<T>, you should also override <, >, <=, >=.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5444000)